experiments/storm/src/main/scala/io/gearpump/experiments/storm/topology/GearpumpTuple.scala

```diff
@@ -18,22 +18,25 @@
 
 package io.gearpump.experiments.storm.topology
 
+import java.lang.{Integer => JInteger}
```

Comment:
In this PR, as a new code reader, I miss the context, why we need to create a name alias for Java primitive types. Can you describe in code comment or somewhere else (e.g. in the PR)